### DISCUSSION TITLE
No NDI Output

### DISCUSSION THREAD

**Discussion by atscomms** [DistroAV/DistroAV#777] [Q&A - Support]
After upgrading from OBS 27 to 28 and then downgrading back to 27.2.4 and reinstalling OBS NDI 4.9.0 with the NDI 4.5 runtime, I can see NDI Output Settings in the Tools menu, I can bring NDI sources into scenes, but there is no NDI output from Main or Preview or from the "Dedicated NDI Output" source filter. I've checked the NDI Analysis on the same machine.
Also, stopping NDI Program or Preview outputs will crash OBS.
This is a Windows 10 PC with an AMD Radeon HD8490 graphics card and with NDI 5 installed.

Once NDI 5 and StreamFX are working on OBS 28 I will switch over, but I'd like to get this working for now. 
Any recommendations would be appreciated.
Thank you

**Comment by Trouffman** [comment]
Closing old thread.
Reason : Using an old plugin/OBS version or old-issue.
